css-two-column-layout/styles.css

```diff
@@ -0,0 +1,82 @@
+body {
+  font-family: Karla, sans-serif;
+}
+
+h2 {
+  text-align: center;
+  font-size: 3.8rem;
+  letter-spacing: 0.3rem;
+  margin-bottom: 30px;
+}
+
+.container {
+  max-width: 1200px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-half {
+  flex-basis: 50%;
+}
+
+.column-one-fourth {
+  flex-basis: 30%;
```

Comment:
This should be `25%`

Reply:
<img width="764" alt="Screen Shot 2022-07-09 at 2 51 11 PM" src="https://user-images.githubusercontent.com/97363006/178123817-13d376f1-99b4-47f7-9c28-9e721681fbec.png">